### DISCUSSION TITLE
High IP abuse reports for Vercel server

### DISCUSSION THREAD

**Discussion by Tadynas** [vercel/vercel#9830] [General] [answered]
### Description

We cannot connect our domain from 3rd party domain provider with Vercel because it says that provided server IP address has active abuse records and provides link to a source: 
https://www.abuseipdb.com/check/76.76.21.21

Is it possible to fix these errors in any way?

Thanks!

**Comment by paulogdm** [comment, marked as answer]
Hello @Tadynas ,

I am a member of our infrastructure team here at Vercel. Any user on the internet can create those reports. As you may know, we use `76.76.21.21` as one of our IPs for our anycast network. Therefore, this IP can be seen in multiple domains hosted on Vercel if you check their DNS records. So it is likely the reports are meant to target Vercel customers, but are being created for Vercel itself.
I would recommend that you first reach out to your 3rd party for clarification and perhaps an exception given the fact above. Another alternative is to use use `cname.vercel-dns.com` (CNAME), as this record we can resolve to IPs in the [range of 76.76.21.0/24](https://whois.arin.net/rest/net/NET-76-76-21-0-1/pft?s=76.76.21.21).

**Reply by Tadynas** [reply]
CNAME record worked, thank you!
